templates/profile/_follow.html

```diff
@@ -0,0 +1,17 @@
+<turbo-frame id="follow-button-{{ target_user.username }}">
+    {% if request.user.is_authenticated %}
+        <form method="POST" action="{% url 'follow_profile' target_user.profile %}">
+            {% csrf_token %}
+            <button class="btn btn-sm btn-outline-secondary action-btn">
```

Comment:
Do you need to change/add/remove a class here based on the follow status?

Reply:
Looking at the [starter kit templates](https://github.com/gothinkster/realworld-starter-kit/blob/master/FRONTEND_INSTRUCTIONS.md), there doesn't seem to be any example the subscribed state for the button. [demo.realworld.io](https://demo.realworld.io) doesn't seem to have any difference either, besides "Follow/Unfollow"

Reply:
On https://demo.realworld.io it does toggle the class from `btn-outline-secondary` to `btn-secondary`. Not a big deal though.